Code/sym.py

```diff
@@ -0,0 +1,34 @@
+# SE HW - 2
+
+class Sym:
+    def __init__(self, column_name, column_position):
+        self.column_name = column_name
```

Comment:
Please use the same variable and function names given in lua file for consistency, like c, s or n, at, name, _has. Using different names will cause a lot of confusion in upcoming homeworks as code gets piled up

Reply:
As discussed, this will be taken up on a different thread.